mockredis/client.py

```diff
@@ -1509,6 +1513,8 @@ def mock_redis_client(**kwargs):
     """
     return MockRedis()
 
+mock_redis_client.from_url = mock_redis_client
```

Comment:
This is so that if you patch `redis.Redis` with `mock_redis_client`, you are still able to call `redis.Redis.from_url()` in the code that is being tested. Without this line, you get an `AttributeError`.